docs/developing/install.rst

```diff
@@ -72,18 +72,6 @@ ps``. You should also be able to visit your Elasticsearch service by opening
 http://localhost:9200/ in a browser, and connect to your PostgreSQL by
 running ``make sql``.
 
-Use pyenv to install Python and tox
------------------------------------
-
-Install Python 3.6 and tox in pyenv:
-
-.. code-block:: shell
-
-   pyenv install 3.6.8
-   pyenv shell 3.6.8
-   pip install tox>=3.8.0
-   pyenv shell --unset
```

Comment:
The old commands install Python, then activate the specified version before installing tox. I can see how the new commands install the desired version, but it doesn't look like they activate it before installing tox?

Reply:
Ok, I've simplified this script by making it unconditionally require pyenv.

I was trying to be clever by making it detect an existing non-pyenv version of Python on your PATH and use that if it's the right version, but that complicates things, introduces edge cases.

So the script now unconditionally tries to use pyenv to install each required version of Python and then installs tox in pyenv's copy of each version of Python. (I moved installing tox out of Makefile into the script, as that makes it possible to ensure that tox is installed in multiple pyenv copies of Python.)

In environments where we don't want pyenv to be used we'll either have to not call this script or make this script detect that environment and abort (as it does for Travis).

You can test it as follows:

Add three versions of Python to `.python-version`:

```diff
diff --git a/.python-version b/.python-version
index 424e1794d..cabddb95a 100644
--- a/.python-version
+++ b/.python-version
@@ -1 +1,3 @@
+3.5.7
 3.6.8
+3.7.4
```

Make sure you don't have any of these versions of Python installed yet:

```
pyenv uninstall -f 3.5.7; pyenv uninstall -f 3.6.8; pyenv uninstall -f 3.7.4
```

Run `make python` and it should install all three versions of Python and install tox in each. To test that it worked:

```
$PYENV_ROOT/versions/3.5.7/bin/tox --version
$PYENV_ROOT/versions/3.6.8/bin/tox --version
$PYENV_ROOT/versions/3.7.4/bin/tox --version
```

Also running `make python` again should do nothing.

If you uninstall tox from one of the versions of Python, and uninstall another Python entirely, then you should see it installing tox in one version of Python, re-installing the removed version of Python and re-installing tox in it, and not doing anything for the untouched version of Python:

```
$PYENV_ROOT/versions/3.6.8/bin/pip uninstall tox
pyenv uninstall -f 3.7.4
make python
```

Afterwards `make python` again should do nothing, and the three `tox --version`s above should work.